frontend/public/co-fetch.js

```diff
@@ -56,6 +62,17 @@ const validateStatus = (response, url) => {
   }
 
   return response.json().then((json) => {
+    // retry 409 conflict errors due to ClustResourceQuota
+    // https://bugzilla.redhat.com/show_bug.cgi?id=1920699
+    if (
+      retry &&
+      method === 'POST' &&
+      response.status === 409 &&
+      _.get(json, 'details.group') === 'quota.openshift.io' &&
+      _.get(json, 'details.kind') === 'clusterresourcequotas'
```

Comment:
I believe this can happen with `resourcequotas` as well. It's just less likely.

Do we need to check group/kind? I'd think a 409 on a `POST` request is always an update to an adjacent resource whether it's quota or something else. In which case, it makes sense to retry regardless of the specific kind.

Reply:
Well interestingly enough I did try with just a `ResourceQuota` but couldn't reproduce. It is an intermittent issue so perhaps I just need to keep trying. I'll investigate further but it was quite reproducible with `ClusterResourceQuota`.
As for retrying only every 409: If you attempt to POST a new resource with a name that already exists, you get a `409`. There is no point in blindly retrying because each subsequent attempt will always fail. Because of this use case, I chose to be more specific.

Reply:
After some more testing I was able to reproduce it with `ResourceQuota`. I will add it as well.

Reply:
@spadgett i've updated the code to also check for `resourcequotas` and dropped the `group` check.